### DISCUSSION TITLE
chat module's intl dependency confict with flutter_localizations

### DISCUSSION THREAD

**Discussion by Frank1016** [serverpod/serverpod#1203] [Ideas]
I was trying to add serverpod_chat_flutter package to my serverpod project's flutter folder, but got this intl dependency conflict error:

```
Resolving dependencies...
Because every version of flutter_localizations from sdk depends on intl 0.18.0 and serverpod_chat_flutter >=1.1.0 depends on intl ^0.18.1, flutter_localizations from sdk is incompatible with serverpod_chat_flutter >=1.1.0.
So, because sbm2025_flutter depends on both flutter_localizations from sdk and serverpod_chat_flutter ^1.1.1, version solving failed.
exit code 1
```

I found a way to override intl dependency in the pubspec.yaml file to resolve the issue, but don't think this is a proper way. So I'm suggesting that maybe it's a good idea to make the intl dependency version of serverpod_chat_flutter package the same with flutter_localizations in the future? Thanks!

**Comment by Isakdl** [comment]
Yes that makes sense, indeed.

**Reply by Frank1016** [reply]
Hi @Isakdl , are you planning to write a Medium article show how to use the chat module from Serverpod just like you did for the auth module? I was trying to learn it from the chat example code only, but couldn't make it work, as there are still a lot things unclear. Or maybe there's other resources that can teach me how to implement the chat module in my serverpod project?
